opentelemetry/chart/templates/logs-collector.yaml

```diff
@@ -56,20 +56,48 @@ spec:
     receivers:
       journald:
         directory: /var/log/journal
+        operators:
+          - id: journal-label
+            type: add
+            field: attributes["log.type"]
+            value: "journald"
 
       k8s_events:
         auth_type: serviceAccount
 
-      filelog:
+      filelog/containerd:
         include_file_path: true
         include: [ /var/log/pods/*/*/*.log ]
+        exclude: [ /var/log/pods/otel_logs-*, /var/log/pods/logs_* ]
         operators:
           - id: container-parser
             type: container
           - id: parser-containerd
             type: add
             field: resource["container.runtime"]
             value: "containerd"
+          - id: container-label
+            type: add
+            field: attributes["log.type"]
+            value: "containerd"
+
+      filelog/files:
+        include: [ /var/log/libvirt/qemu/*.log, /var/log/openvswitch/*.log ]
+        include_file_path: true
+        start_at: beginning
+        multiline:
+          line_start_pattern: ^\d{4}-\d{2}-\d{2}
+        operators:
+          - type: regex_parser
+            regex: (?P<logtime>\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3})
+          - type: time_parser
+            parse_from: attributes.logtime
+            layout: '%Y-%m-%dT%H:%M:%S.%L'
+            layout_type: strptime
+          - id: file-label
+            type: add
+            field: attributes["log.type"]
+            value: "files"
```

Comment:
Line 92: Why is a named capture group being used here? Can you reference it somewhere else? 
Line 94: Is it being used there directly or is that some other `logtime`?

Reply:
The time_parser cannot access the body, one has to define a parser first and afterwards access the field: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/291159720ba75185c185e0e8ba1cceb6a5453d52/receiver/filelogreceiver

Reply:
Ah I see!